app/static/cursor.js

```diff
@@ -0,0 +1,41 @@
+CURSORS_PATH = "/media/cursors/";
+window.addEventListener("load", get_cursor_choices);
+
+/**
+ * @summary This function gets cursor choices from the db.
+ */
+function get_cursor_choices() {
+  let request = new XMLHttpRequest();
+  request.open("GET", "/cursor/load_cursor", true);
+  request.onload = change_cursor;
+  request.send();
+}
+
+/**
+ * @summary This function changes the primary cursor and the secondary
+ * cursor according to users' choices.
+ */
+function change_cursor() {
+  let cursor_settings = JSON.parse(JSON.parse(this.response));
+  let primary_cursor = cursor_settings["primary_cursor"];
+  let primary_cursor_val =
+    "url(" + CURSORS_PATH + primary_cursor + ")" + ", auto";
```

Comment:
you can use `${strings} `` format for JS strings

Reply:
Nice! I thought it was only in jquery. Thank you!